kratos/python_scripts/testing_utilities.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import print_function, absolute_import, division
+
+# Import Kratos
+import KratosMultiphysics
+try:
+    import KratosMultiphysics.mpi as KratosMPI
+    rank = KratosMPI.mpi.rank
+except ImportError:
+    rank = 0
+
+# Other imports
+import os
+
+def DeleteFileIfExisting(file_name):
+    if rank == 0:
+        try:
+            os.remove(file_name)
+        except FileNotFoundError:
+            pass
```

Comment:
os.remove can also throw IsADirectoryError.

Reply:
You can also use os.path.isfile(file_name).

Reply:
thx for the hint